spot_wrapper/wrapper.py

```diff
@@ -443,7 +445,7 @@ def __init__(
                 self._spot_check_client = self._robot.ensure_client(SpotCheckClient.default_service_name)
                 self._mission_client = self._robot.ensure_client(MissionClient.default_service_name)
                 self._license_client = self._robot.ensure_client(LicenseClient.default_service_name)
-                if self._robot.has_arm():
+                if self._robot.has_arm() and not self.gripperless:
```

Comment:
You should swap these conditions. `has_arm` involves a grpc call.

Reply:
good point